suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/AbstractSpdzTest.java

```diff
@@ -51,32 +58,38 @@
   protected void runTest(
       TestThreadRunner.TestThreadFactory<SpdzResourcePool, ProtocolBuilderNumeric> f,
       EvaluationStrategy evalStrategy,
-      PreprocessingStrategy preProStrat, int noOfParties) throws Exception {
+      PreprocessingStrategy preProStrat, int noOfParties) {
     runTest(f, evalStrategy, preProStrat, noOfParties, false);
   }
 
   protected void runTest(
       TestThreadRunner.TestThreadFactory<SpdzResourcePool, ProtocolBuilderNumeric> f,
       EvaluationStrategy evalStrategy,
-      PreprocessingStrategy preProStrat, int noOfParties, boolean logPerformance)
-      throws Exception {
+      PreprocessingStrategy preProStrat, int noOfParties, boolean logPerformance) {
     List<Integer> ports = new ArrayList<>(noOfParties);
     for (int i = 1; i <= noOfParties; i++) {
       ports.add(9000 + i * (noOfParties - 1));
     }
 
+    int maxBitLength;
```

Comment:
The 150 doesn't matter for other preprocessing types. Just make it 128 if this is required by MASCOT

Reply:
The bitlength has relation to the size of modulus but also needs to be aligned to 8.

Let us fix the mim test and then see